packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -16,22 +16,12 @@ export default {
         disable: true,
       },
     },
-    onclose: {
-      description:
-        'If closeable is true, this event is triggered when an alert is closed.',
-      table: {
-        defaultValue: {
-          detail: undefined,
-        },
-      },
-      control: false,
-    },
   },
   parameters: {
     docs: {
       description: {
         component: `Use a heading element with an attribute named slot and a value of "headline" to control what is displayed for the alert's headline. 
-        Any children passed into this component without a parent slot "headline" will render in the alert's body.`,
+        Any children passed into this component without a parent slot "headline" will render in the alert's body. This component has 1 event: onClose and onVaComponentDidLoad. If you need to use these events in React applications, please use the bindings import for this web component. See https://design.va.gov/documentation/developers#react-applications.`,
```

Comment:
@andresriveratoro think a typo - onClose and onVaComponentDidLoad 2 events

Reply:
Good catch, thanks!